Fix implicit declaration of function 'IoState_registerProtoWithFunc_'
and incompatible pointer types.
https://github.com/IoLanguage/io/pull/387
https://github.com/IoLanguage/io/commit/ad58e1c661874e779f3638bb19090d633badb715
--- addons/EditLine/source/IoEditLine.c.orig
+++ addons/EditLine/source/IoEditLine.c
@@ -56,7 +56,7 @@ IoEditLine *IoEditLine_proto(void *state)
 
 	el_source(DATA(self)->editline, NULL);
 
-	IoState_registerProtoWithFunc_((IoState *)state, self, IoEditLine_proto);
+	IoState_registerProtoWithFunc_((IoState *)state, self, (char *) IoEditLine_proto);
 
 	IoObject_addMethodTable_(self, methodTable);
 
--- libs/iovm/source/IoState.h.orig
+++ libs/iovm/source/IoState.h
@@ -175,6 +175,7 @@ IOVM_API void IoState_init(IoState *self);
 void IoState_setupQuickAccessSymbols(IoState *self);
 void IoState_setupCachedMessages(IoState *self);
 void IoState_setupSingletons(IoState *self);
+void IoState_registerProtoWithFunc_(IoState *self, IoObject *proto, const char *v);
 
 // setup tags
 
